taxcalc/utils.py

```diff
@@ -428,8 +445,48 @@ def create_distribution_table(calc, groupby, result_type):
         gp_mean = weighted_avg_allcols(df, TABLE_COLUMNS)
         sum_row = get_sums(df, na=True)[TABLE_COLUMNS]
 
-    return gp_mean.append(sum_row)
+    if groupby == "everyone":
+        return gp_mean.append(sum_row).ix[1:]
+    else:
+        return gp_mean.append(sum_row)
+
+
+def create_diagnostic_table(calc, num_years=12):
+    """
+    Create a table of important tax variables over multiple years. 
+    Does not mutate the calculator. 
+
+    Parameters
+    ----------
+    calc : Calculator to create a diagnostic table from. 
+
+    num_years : Number of years to include in the diagnostic table. 
+                Default is set to 12 to include 2013 to 2024.
+
+    Returns
+    -------
+    DataFrame where columns are years and rows are aggregated tax variables. 
+    """
 
+    calc = copy.deepcopy(calc)
```

Comment:
Could be reasonable to allow the caller to specify whether or not to mutate the `calc` object. So something like a keyword argument `make_copy=True`, so that way if you don't mind messing up the `calc` object to make the table you don't have to pay for a copy. Just a suggestion though. I don't feel strongly.